shimmingtoolbox/load_nifti.py

```diff
@@ -151,36 +152,99 @@ def load_nifti(path_data, modality='phase'):
     return niftis, info, json_info
 
 
-def read_nii(nii_path, auto_scale=True):
+def read_nii(fname_nifti, auto_scale=True):
     """ Reads a nifti file and returns the corresponding image and info. Also returns the associated json data.
     Args:
-        nii_path (str): direct path to the .nii or .nii.gz file that is going to be read
+        fname_nifti (str): direct path to the .nii or .nii.gz file that is going to be read
         auto_scale (:obj:`bool`, optional): Tells if scaling is done before return
     Returns:
         info (Nifti1Image): Objet containing various data about the nifti file (returned by nibabel.load)
         json_data (dict): Contains the different fields present in the json file corresponding to the nifti file
-        image (ndarray): Image contained in the read nifti file. Siemens phase images are rescaled between 0 and 2pi.
+        image (ndarray): For B0-maps, image contained in the nifti. Siemens phase images are rescaled between 0 and 2pi.
+                         For RF-maps, complex array of dimension (x, y, slice, coil) with phase between -pi and pi.
```

Comment:
```suggestion
        image (ndarray): For B0-maps, image contained in the nifti. Siemens phase images are rescaled between 0 and 2pi.
        For RF-maps, complex array of dimension (x, y, slice, coil) with phase between -pi and pi.
```
@gaspardcereza I think it might be this indentation. You can apply the change from this suggestion directly.

Reply:
The warning in question triggering this is in the RTD build log:
`/home/docs/checkouts/readthedocs.org/user_builds/shimming-toolbox-py/checkouts/235/shimmingtoolbox/load_nifti.py:docstring of shimmingtoolbox.load_nifti.read_nii:10: WARNING: Unexpected indentation.`

Reply:
Oh yes you're right I missed it because it was too far right out of my screen 😅 Thank you !